source/support/spawn_vehicle.sqf

```diff
@@ -21,11 +21,9 @@ if (commandpointsblu1 >= _cost) then {
 	hint "Vehicle ready! check map for location";
 	commandpointsblu1 = commandpointsblu1 - _cost;
 	ctrlSetText [1000, format["%1",commandpointsblu1]];
-	_vehic = _name createVehicle [0,0,0];

```

Comment:
according to https://community.bistudio.com/wiki/Code_Optimisation
createVehicle(Local)
createVehicle(Local) position is not exact so you must use setPos but this is very slow, to create the object on [0,0,0] and then set the position is faster.

_obj = 'Land_Stone_4m_F' createVehicle [0,0,0]; //also createVehicleLocal
_obj setPos (getPos player); //0,03ms (100 testcycles)
is 200 times faster than...

_obj = 'Land_Stone_4m_F' createVehicle (getPos player); //also createVehicleLocal
_obj setPos (getPos player); //5,9ms (100 testcycles)